cub/device/device_radix_sort.cuh

```diff
@@ -66,9 +66,9 @@ namespace cub {
  * half-precision floating-point type.  Although the direct radix sorting
  * method can only be applied to unsigned integral types, DeviceRadixSort
  * is able to sort signed and floating-point types via simple bit-wise transformations
- * that ensure lexicographic key ordering. For floating-point types -0.0 and +0.0 are
- * considered equal and appear in the result in the same order as they appear in
- * the input.
+ * that ensure lexicographic key ordering. DeviceRadixSort is stable. For
+ * floating-point types -0.0 and +0.0 are considered equal and appear in the
+ * result in the same order as they appear in the input.
```

Comment:
Can we also document that cub does not support in-place operations, so input and output pointers can not have memory overlap?

Reply:
Absolutely, I'll add that before this goes in. 👍